doc/api/dgram.md

```diff
@@ -561,7 +561,7 @@ s.bind(1234);
 s.addMembership('224.0.0.114');
 ```
 
-Must be changed to pass a callback function to the [`dgram.Socket#bind()`][]
+must be changed to pass a callback function to the [`dgram.Socket#bind()`][]
```

Comment:
Rather than continuing the sentence after the example, it would probably be better to split it into two proper sentences. Not a blocking objection, though. Just an optional nit. (Maybe instead of `Legacy code that assumes synchronous behavior, as in the following example:` could be changed to `Legacy code would use syncrhronous behavior:` and line 564 can be changed to `Such legacy code would need to be changed to pass a ...`

Alternative: Given that this is about a change that appeared in Node.js v0.10, I wonder if it can be removed at this time? It's only going to be relevant to people who are porting v0.8.x code and earlier.